### ISSUE TITLE
when I delete the " autocmd vimEnter * NERDTree",there is something wrong,why?how can I fix it?

### ISSUE THREAD

**Issue by BearJean** [scrooloose/nerdtree#823]
### Environment

* Operating System: 
* Vim version `:version`: 
* NERDTree version `git rev-parse --short HEAD`: 
* NERDTree settings applied in your vimrc, if any:
    ```vim
    ```

### Process

1. 

### Possible Fix

here is the bug output:
Error detected while processing function <SNR>44_IndentGuidesEnable[1]..indent_guides#enable[8]..indent_guides#init_script_vars[7]..indent_guides#capture_highlight:
line    2:
E411: highlight group not found: Normal
Press ENTER or type command to continue
Error detected while processing function <SNR>44_IndentGuidesEnable[1]..indent_guides#enable[8]..indent_guides#init_script_vars[7]..indent_guides#capture_highlight[2]..<SNR>
97_PollServerReady[14]..<SNR>97_OnFileTypeSet[10]..<SNR>97_OnFileReadyToParse:
line    9:
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/home/tron/.vim/bundle/YouCompleteMe/autoload/../python/ycm/youcompleteme.py", line 454, in OnFileReadyToParse
    self._AddSyntaxDataIfNeeded( extra_data )
  File "/home/tron/.vim/bundle/YouCompleteMe/autoload/../python/ycm/youcompleteme.py", line 869, in _AddSyntaxDataIfNeeded
    syntax_parse.SyntaxKeywordsForCurrentBuffer() )
  File "/home/tron/.vim/bundle/YouCompleteMe/autoload/../python/ycm/syntax_parse.py", line 72, in SyntaxKeywordsForCurrentBuffer
    syntax_output = vimsupport.CaptureVimCommand( 'syntax list' )
  File "/home/tron/.vim/bundle/YouCompleteMe/autoload/../python/ycm/vimsupport.py", line 175, in CaptureVimCommand
    vim.command( 'redir => b:ycm_command' )
vim.error: Vim(redir):E121: Undefined variable: l:output
E171: Missing :endif

**Comment by PhilRunninger** [comment]
You've provided practically no information to help debug this issue, and nothing is specific to NERDTree in your text. Nevertheless, I took a look at your .vimrc file(s), and I do have a suggestion for you. Between
```
filetype off                  " required
" set the runtime path to include Vundle and initialize
set rtp+=~/.vim/bundle/Vundle.vim
call vundle#begin()
```
and
```
call vundle#end()            " required
filetype plugin indent on    " required
```
you have a mix of `Plugin ...` statements and other configuration statements. I've discovered that sometimes that can cause errors to occur. Try keeping only the `Plugin ...` statements in that block, and move the other configuration statements after the `filetype plugin indent on` line.
